recipes/muparserx/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "muparserx" %}
+{% set version = "4.0.8" %}
+
+package:
+  name: muparserx
+  version: {{ version }}
+
+source:
+  url: https://github.com/beltoforion/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 5913e0a4ca29a097baad1b78a4674963bc7a06e39ff63df3c73fbad6fadb34e1
+
+build:
+  number: 0
+  skip: True  # [win]
+  run_exports:
+    - {{ pin_subpackage('muparserx', max_pin='x.x') }}
+
+requirements:
+  build:
+    - cmake
+    - make  # [unix]
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - llvm-openmp  # [osx]
+  run:
+    - llvm-openmp  # [osx]
+test:
+  commands:
+    - test -f ${PREFIX}/include/muparserx/mpParser.h  # [unix]
+    - test -f ${PREFIX}/lib/libmuparserx${SHLIB_EXT}  # [unix]
```

Comment:
Please existence tests for a CMake and package config file to indicate that they are available.